src/pudl/metadata/resources/eia860.py

```diff
@@ -802,6 +802,47 @@
         "sources": ["eia860"],
         "etl_group": "eia860",
     },
+    "core_eia860__yearly_generators_energy_storage": {
```

Comment:
This is another table that seems like it might currently be a mix of slowly changing dimensions and yearly data. Do the boolean values really change every year in general? Or are they stable descriptors of what role the battery plays longer term?

Reply:
I just did a little review and it looks like the boolean values are very stable SCD values. Should I turn this into two tables? an SCD and a yearly table?

Reply:
It looks like the other non-boolean columns are also mostly stable, but not as stable as the booleans. What is the threshold for SCD?

Reply:
we don't have a set threshold but if most of the columns are mostly stable i'd say go for it on the scd name